highfidelity/hifi
# Fixed an issue where cursor repeatedly is made visible

Should fix https://highfidelity.fogbugz.com/f/cases/4977/Reticle-setVisible-no-longer-works

Test Plan:

1. Launch interface
2. Verify Reticle.setVisible works (used in gracefulControls.js, for example) as intended
3. Verify cursor/reticle is hidden when it should be
